apps/base/utils.py

```diff
@@ -69,3 +74,23 @@ def decode_data(data):
     for i in data:
         decoded.append(base64.decodestring(i+"=="))
     return decoded
+
+
+def send_slack_notification(webhook=settings.SLACK_WEBHOOKS['default'], message=""):
+    '''Send slack notification to any workspace
+
+    Keyword Arguments:
+        webhook {string} -- slack webhook URL (default: {settings.SLACK_WEBHOOKS['default']})
+        message {str} -- JSON/Text message to be sent to slack (default: {""})
+    '''
+    if settings.DEBUG is False:
+        try:
+            response = requests.post(
+                webhook,
+                data=json.dumps({'text': str(message)}),
+                headers={'Content-Type': 'application/json'}
+            )
+            return response
+        except Exception as e:
+            logger.info(
+                'Exception raised while sending slack notification. \n Exception message: {}'.format(e))
```

Comment:
Please send the slack message in logger.info if the request didn't go through properly.